gobblin-runtime/src/test/java/org/apache/gobblin/runtime/api/MysqlMultiActiveLeaseArbiterTest.java

```diff
@@ -38,6 +38,7 @@ public class MysqlMultiActiveLeaseArbiterTest {
   private static final int EPSILON = 10000;
   private static final int MORE_THAN_EPSILON = (int) (EPSILON * 1.1);
   private static final int LINGER = 50000;
+  private static final int MORE_THAN_LINGER = (int) (LINGER * 1.1);
```

Comment:
ideally it should be typecasted to `long` because sleep takes `long`. same with `MORE_THAN_EPSILON`